src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,25 +29,39 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              Todo #{todo.id}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-            <button type="button" className="delete" data-cy="modal-close" />
+            <button
+              type="button"
+              className="delete"
+              data-cy="modal-close"
+              onClick={onClose}
+            />
           </header>
 
           <div className="modal-card-body">
             <p className="block" data-cy="modal-title">
-              quis ut nam facilis et officia qui
+              {todo.title}
             </p>
 
             <p className="block" data-cy="modal-user">
               {/* <strong className="has-text-success">Done</strong> */}
-              <strong className="has-text-danger">Planned</strong>
+              <strong
+                className={
+                  todo.completed ? 'has-text-success' : 'has-text-danger'
```

Comment:
Consider using the `classnames` library for conditional class names here. This can improve readability and maintainability of your code, as specified in the checklist.

Reply:
> Consider using the `classnames` library for conditional class names here. This can improve readability and maintainability of your code, as specified in the checklist.

Good point